tests/foreman/ui/test_syncplan.py

```diff
@@ -83,14 +39,29 @@ def test_positive_create_basic(self, test_data):
 
         """
         with Session(self.browser) as session:
-            make_syncplan(
-                session,
-                org=self.organization.name,
-                name=test_data['name'],
-                description=test_data['desc'],
-                sync_interval=test_data['interval'],
-            )
-            self.assertIsNotNone(self.syncplan.search(test_data['name']))
+            for value in generate_strings_list():
```

Comment:
Previous coverage:
```
    @data(
        {
            u'name': gen_string('alpha'),
            u'desc': gen_string('alpha'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name': gen_string('numeric'),
            u'desc': gen_string('numeric'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name': gen_string('alphanumeric'),
            u'desc': gen_string('alphanumeric'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name': gen_string('utf8'),
            u'desc': gen_string('utf8'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name': gen_string('html', 20),
            u'desc': gen_string('html'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name': gen_string('alphanumeric'),
            u'desc': gen_string('alphanumeric'),
            u'interval': SYNC_INTERVAL['day']
        },
        {
            u'name': gen_string('utf8'),
            u'desc': gen_string('utf8'),
            u'interval': SYNC_INTERVAL['day']
        },
        {
            u'name': gen_string('alpha'),
            u'desc': gen_string('alpha'),
            u'interval': SYNC_INTERVAL['week']
        },
        {
            u'name': gen_string('utf8'),
            u'desc': gen_string('utf8'),
            u'interval': SYNC_INTERVAL['week']
        },
        {
            u'name': gen_string('html'),
            u'desc': gen_string('html'),
            u'interval': SYNC_INTERVAL['week']
        }
    )
```

New coverage:
```
    @data(
        {
            u'name' = u'desc': gen_string('alpha'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name' = u'desc': gen_string('numeric'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name' = u'desc': gen_string('alphanumeric'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name' = u'desc': gen_string('utf8'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name' = u'desc': gen_string('html'),
            u'interval': SYNC_INTERVAL['hour']
        },
        {
            u'name' = u'desc': gen_string('alphanumeric'),
            u'interval': SYNC_INTERVAL['day']
        },
        {
            u'name' = u'desc': gen_string('alphanumeric'),
            u'interval': SYNC_INTERVAL['week']
        },
    )
```

So we have some differences in coverage. I pretty don't care how many methods we have, but not sure that like mentioned difference